### PR TITLE
Update to 1.0.1

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #18 

Generated with Grayskull

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
   host:
     - pip
-    - python
-    - setuptools
-
+    - python >=3.5
   run:
-    - python
     - nose
-    - termcolor
```
Comment (jakirkham) [inline recipe/meta.yaml:26]: Can we please keep `termcolor`? It is [optional]( https://github.com/mahmoudimus/nose-timer/blob/aabc31c2687c5fe0031dab6a781e9b3070472c29/nosetimer/plugin.py#L16-L19 ), but useful for seeing what is faster or slower.

Also may be worth [adding `colorama`]( https://github.com/mahmoudimus/nose-timer/blob/aabc31c2687c5fe0031dab6a781e9b3070472c29/nosetimer/plugin.py#L21-L29 ) as well

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
 package:
-  name: nose-timer
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: nose-timer-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/n/nose-timer/nose-timer-{{ version }}.tar.gz
-  sha256: 4c383d67ae6affd67173217e23689308c1a2bb178bfaf8b05b06c4cf16971c26
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nose-timer-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipe/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipe/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipe/meta.yaml:26]: Ahh yeah sorry one sec.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Bastian! 😄

**Comment by BastianZim** [comment]
Thanks as well John!

**Pull request merged**